<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgressSettingsForm</class>
 <widget class="QWidget" name="ProgressSettingsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>240</width>
     <height>197</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="disabledLabel">
      <property name="text">
       <string>Disabled</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QCheckBox" name="disabledCheckBox"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="progressTypeLabel">
      <property name="text">
       <string>Progress type</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="progressTypeComboBox">
      <item>
       <property name="text">
        <string>Determinate</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Indeterminate</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="progressLabel">
      <property name="text">
       <string>Progress</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSlider" name="progressSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="useThemeColorsLabel">
      <property name="text">
       <string>Use theme colors</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QCheckBox" name="useThemeColorsCheckBox"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="colorLabel">
      <property name="text">
       <string>Progress color</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="progressColorLineEdit"/>
      </item>
      <item>
       <widget class="QToolButton" name="progressColorToolButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="backgroundColorLabel">
      <property name="text">
       <string>Background color</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="backgroundColorLineEdit"/>
      </item>
      <item>
       <widget class="QToolButton" name="backgroundColorToolButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
